src/main/java/stream/flarebot/flarebot/util/ReportManager.java

```diff
@@ -0,0 +1,62 @@
+package stream.flarebot.flarebot.util;
+
+import org.joda.time.DateTime;
+import stream.flarebot.flarebot.objects.Report;
+import stream.flarebot.flarebot.objects.ReportStatus;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportManager {
+
```

Comment:
I wouldn't. If the entries won't be unique use `ConcurrentHashMap#newKeySet()`. It's way more efficient, and provides uniqueness. Keep in mind hashCode needs to be implemented. Use Alt+Insert then hashCode() and equals()